docs/markdown/Python-module.md

```diff
@@ -238,6 +238,63 @@ module.
 
 It exposes the same methods as its parent class.
 
+## A note about Debian installation paths
+
+Debian and derivatives (Ubuntu, Linux Mint etc.) [change default installation
+and import paths][debian-dist-packages] in system interpreter. The problem is,
+the way they do this is inconsistent with documented ways of discovering those
+paths inside Python. Specifically, the [`sysconfig` module][python-sysconfig]
+returns values that are not in [`sys.path`][python-sys-path]. This module is
+used internally by Meson and therefore the installation paths . But it is
+generally expected that if you configure build with `--prefix=/usr` or
+`--prefix=/usr/local` (the latter is the default), Python modules should be
+available without adjusting [`$PYTHONPATH`][python-pythonpath]. Therefore,
+a&nbsp;workaround is needed. There are two, non mutually exclusive options:
```

Comment:
why `&nbsp;` here?
'non-mutually' should be hypenated. 
putting it in a postive form ("two solutions, which can be used in combination") might be clearer?

Reply:
This is still not fixed, please fix the formatting.